includes/reader-revenue/woocommerce/my-account/class-my-account-ui-v1.php

```diff
@@ -0,0 +1,101 @@
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Newspack "My Account" customizations v1.x.x.
+ */
+class My_Account_UI_V1 extends WooCommerce_My_Account {
+	/**
+	 * Initialize.
+	 *
+	 * @codeCoverageIgnore
+	 */
+	public static function init() {
+		\add_filter( 'body_class', [ __CLASS__, 'add_body_class' ] );
+		\add_filter( 'do_shortcode_tag', [ __CLASS__, 'add_newspack_ui_wrapper' ], 10, 2 );
+		\add_action( 'wp_enqueue_scripts', [ __CLASS__, 'enqueue_assets' ], 11 );
+		\add_filter( 'wc_get_template', [ __CLASS__, 'wc_get_template' ], 10, 5 );
+	}
+
+	/**
+	 * Add a body class to the My Account page.
+	 *
+	 * @param array $classes The body classes.
+	 * @return array The body classes.
+	 */
+	public static function add_body_class( $classes ) {
+		if ( function_exists( 'is_account_page' ) && \is_account_page() ) {
+			$classes[] = 'newspack-my-account';
+			$classes[] = 'newspack-my-account--v1';
+			if ( ! \is_user_logged_in() ) {
+				$classes[] = 'newspack-my-account--logged-out';
+			} else {
+				$classes[] = 'newspack-my-account--logged-in';
+			}
+		}
+		return $classes;
+	}
+
+	/**
+	 * Render a wrapper element to apply Newspack UI styles to My Account page content.
+	 *
+	 * @param string $output The output.
+	 * @param string $tag The tag.
+	 *
+	 * @return string The output.
+	 */
+	public static function add_newspack_ui_wrapper( $output, $tag ) {
+		if ( 'woocommerce_my_account' === $tag ) {
+			return '<div class="newspack-ui">' . $output . '</div>';
+		}
+		return $output;
+	}
+
+	/**
+	 * Enqueue assets.
+	 */
+	public static function enqueue_assets() {
+		if ( function_exists( 'is_account_page' ) && \is_account_page() ) {
+			// Dequeue styles from the Newspack theme first, for a fresh start.
+			\wp_dequeue_style( 'newspack-woocommerce-style' );
+			\wp_enqueue_style(
+				'my-account-v1',
+				\Newspack\Newspack::plugin_url() . '/dist/my-account-v1.css',
+				[],
+				NEWSPACK_PLUGIN_VERSION
+			);
+		}
+	}
+
+	/**
+	 * WC's page templates hijacking.
+	 *
+	 * @param string $template      Template path.
+	 * @param string $template_name Template name.
+	 */
+	public static function wc_get_template( $template, $template_name ) {
+		switch ( $template_name ) {
+			case 'myaccount/form-login.php':
+				if ( isset( $_GET[ self::AFTER_ACCOUNT_DELETION_PARAM ] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+					return dirname( NEWSPACK_PLUGIN_FILE ) . '/includes/reader-revenue/woocommerce/my-account/templates/myaccount-after-delete-account.php';
+				}
+				return $template;
+			case 'myaccount/form-edit-account.php':
+				if ( isset( $_GET[ self::DELETE_ACCOUNT_FORM ] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+					return dirname( NEWSPACK_PLUGIN_FILE ) . '/includes/reader-revenue/woocommerce/my-account/templates/myaccount-delete-account.php';
+				}
+				return dirname( NEWSPACK_PLUGIN_FILE ) . '/includes/reader-revenue/woocommerce/my-account/templates/myaccount-edit-account.php';
```

Comment:
Same as the previous comment, I think we can use just `__DIR__`

Reply:
Update in ae456e80